public/app/plugins/panel/alertmanager-notifications/AMNotifications.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { PanelProps } from '@grafana/data';
+import { CustomScrollbar } from '@grafana/ui';
+
+import { AlertmanagerGroup, Matcher } from 'app/plugins/datasource/alertmanager/types';
+import { fetchAlertGroupsAction } from 'app/features/alerting/unified/state/actions';
+import { initialAsyncRequestState } from 'app/features/alerting/unified/utils/redux';
+import { NOTIFICATIONS_POLL_INTERVAL_MS } from 'app/features/alerting/unified/utils/constants';
+import { useUnifiedAlertingSelector } from 'app/features/alerting/unified/hooks/useUnifiedAlertingSelector';
+
+import { AmNotificationsGroup } from './AmNotificationsGroup';
+import { AMNotificationsOptions } from './types';
+import { parseMatchers } from 'app/features/alerting/unified/utils/alertmanager';
+import { useFilteredGroups } from './useFilteredGroups';
+
+export const AMNotifications = (props: PanelProps<AMNotificationsOptions>) => {
+  const dispatch = useDispatch();
+
+  const alertManagerSourceName = props.options.alertmanager;
+
+  const alertGroups = useUnifiedAlertingSelector((state) => state.amAlertGroups) || initialAsyncRequestState;
+  const loading = alertGroups[alertManagerSourceName || '']?.loading;
+  const error = alertGroups[alertManagerSourceName || '']?.error;
+  const results: AlertmanagerGroup[] = alertGroups[alertManagerSourceName || '']?.result || [];
+  const matchers: Matcher[] = props.options.labels ? parseMatchers(props.options.labels) : [];
+
+  const filteredResults = useFilteredGroups(results, matchers);
+
+  useEffect(() => {
+    function fetchNotifications() {
+      if (alertManagerSourceName) {
+        dispatch(fetchAlertGroupsAction(alertManagerSourceName));
+      }
+    }
+    fetchNotifications();
+    const interval = setInterval(() => fetchNotifications, NOTIFICATIONS_POLL_INTERVAL_MS);
```

Comment:
```suggestion
    const interval = setInterval(fetchNotifications, NOTIFICATIONS_POLL_INTERVAL_MS);
```
does not invoke `fetchNotifications`, so polling does not work